### PR TITLE
Okteto login depreciation message

### LINKED ISSUE #1922: Add deprecation message to `okteto namespace`
**Is your feature request related to a problem? Please describe.**
`okteto namespace` will be deprecated in favor of `okteto context update-kubeconfig`.
We should add a `warning` message with a deprecation notice if people run `okteto namespace`

### PR DESCRIPTION
**Pull request by Youngestdev**
Fixes #1922 

## Proposed changes

- Added a message to inform users to switch to using `okteto context` in place of `okteto login` as it will be depreciated soon.
-

### CODE REVIEW

File: cmd/login.go [hunk 1]
```diff
@@ -25,6 +26,7 @@ import (
 
 // Login starts the login handshake with GitHub and okteto
 func Login() *cobra.Command {
+    log.Information("okteto login will soon be depreciated, run okteto context to configure your cluster.")
```
Comment (jLopezbarb) [inline cmd/login.go:29]: I would use `log.Warning`. It will set the color to yellow which is the standard for warnings

Comment (ifbyol) [reply cmd/login.go:29]: Also, the indentation doesn't seem right, at least, I see it a bit to the left

Comment (jLopezbarb) [reply cmd/login.go:29]: We also surround okteto commands with '' so the user knows where the command starts and ends. 
`okteto login will soon be depreciated, run okteto context to configure your cluster.` -> `'okteto login' will soon be depreciated, run 'okteto context' to configure your cluster.`

Comment (ifbyol) [reply cmd/login.go:29]: This is what I mean, it seems `log....` is not aligned with the next line, at least this is how I see it in GitHub, not sure if it is right in the code or not

Captura de pantalla 2021-11-04 a las 11 26 50

Comment (Youngestdev) [reply cmd/login.go:29]: Yep. I thought of this. I'll change this.

Comment (Youngestdev) [reply cmd/login.go:29]: Oh @ifbyol, interestingly, it's on the same line in my IDE 😬

File: cmd/login.go [hunk 2]
```diff
@@ -25,6 +26,7 @@ import (
 
 // Login starts the login handshake with GitHub and okteto
 func Login() *cobra.Command {
+	log.Warning("'okteto login' will soon be depreciated, run 'okteto context' to configure your cluster.")
```
Comment (rberrelleza) [inline cmd/login.go:29]: okteto context configures the access to the cluster, not the actual cluster. Suggestion below. 

```suggestion
	log.Warning("'okteto login' will soon be deprecated in favor of 'okteto context'.")
```

**Review by ifbyol — APPROVED** [review]

File: cmd/login.go [hunk 3]
```diff
@@ -25,6 +26,7 @@ import (
 
 // Login starts the login handshake with GitHub and okteto
 func Login() *cobra.Command {
+	log.Warning("'okteto login' will soon be deprecated in favor of 'okteto context'.")
```
Comment (pchico83) [inline cmd/login.go:29]: I would add a link to:
https://okteto.com/docs/reference/cli/#context

And I would say it's is deprecated in favor of `okteto context`. And it won't be available anymore from 1.16+

**Review by pchico83 — APPROVED** [review]

**Comment by pchico83** [comment]
@Youngestdev could you finish this one as part of December planning?

**Comment by Youngestdev** [comment]
Sure @pchico83

File: cmd/login.go (hunk 3 above)
Comment (Youngestdev) [reply cmd/login.go:29]: Oh, great.

**Review by jLopezbarb — APPROVED** [review]

File: cmd/login.go [hunk 4]
```diff
@@ -26,6 +28,7 @@ import (
 
 // Login starts the login handshake with GitHub and okteto
 func Login() *cobra.Command {
+	log.Warning("'okteto login' will soon be deprecated in favor of 'okteto context' and won't be available from 1.16+. Check out okteto context: https://okteto.com/docs/reference/cli/#context")
```
Comment (pchico83) [inline cmd/login.go:31]: @rberrelleza could you review this message?

Comment (rberrelleza) [inline cmd/login.go:31]:
```suggestion
	log.Warning("'okteto login' will soon be deprecated in favor of 'okteto context', and will be removed in version 1.16.  Learn more about okteto context at https://okteto.com/docs/reference/cli/#context")
```

**Pull request merged**